rethinkdb/content.md

```diff
@@ -0,0 +1,46 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance allowing connections on all interfaces
+
+```bash
+docker run --name some-rethink -d rethinkdb --bind all
+```
+
+## connect to it from an application ...
+
+```bash
+docker run --name some-app --link some-rethink:rdb -d application-that-uses-rdb
+```
+
+## ... via the web admin interface ...
+
+```bash
+# browsing on the same machine:
+xdg-open \
+  "http://$(docker inspect --format '{{ .NetworkSettings.IPAddress }}'):8080"
+
+# exposing and connecting remotely:
+docker run -d --link some-rethink:rdb -p 8080:8080 svendowideit/ambassador
```

Comment:
No, that's supposed to be ambassador. It runs `socat` to forward ports from the running RethinkDB instance to the exposed host port (the ambassador pattern).